clients/lsp-eslint.el

```diff
@@ -233,8 +233,10 @@ source.fixAll code action."
   (lsp-stdio-connection
    (lambda () lsp-eslint-server-command)
    (lambda ()
-     (and (cl-second lsp-eslint-server-command)
-          (file-exists-p (cl-second lsp-eslint-server-command)))))
+    (let* ((command-name (f-base (f-filename (cl-first lsp-eslint-server-command))))
+          (first-argument (cl-second lsp-eslint-server-command))
+          (first-argument-exist (and first-argument (file-exists-p first-argument))))
+     (if (equal command-name "node") first-argument-exist t))))
```

Comment:
if the first element is not `node`, then we should check if the first element exists in the `else` block, right?

Reply:
how can I check that? the command I'm expecting is something like `eslint-server` instead of the path to the command.

Reply:
I think that you should do something like `(executable-find (cl-first (lsp-eslint-server-command)))` in the else block.

Reply:
done!